addons/mail/static/tests/chat_bubble/chat_bubble.test.js

```diff
@@ -46,6 +46,7 @@ test("No duplicated chat bubbles", async () => {
     await click(".o_menu_systray i[aria-label='Messages']");
     await click(".o-mail-MessagingMenu button", { text: "New Message" });
     await insertText("input[placeholder='Search a conversation']", "John");
+    await contains(".o_command_name", { count: 3 });
```

Comment:
Should we maybe have an explicit count before input to see the value before/after so we properly ensure a change in all tests?

Reply:
Also the test might be even more "stable" if we wait for initial results before using input.